lms/templates/student_account/account_settings.underscore

```diff
@@ -1,26 +1,18 @@
 <main id="main" aria-label="Content" tabindex="-1">
 <div class="account-settings-container">
     <div class="wrapper-header">
-        <h2 class="header-title"><%- gettext("Account Settings") %></h2>
-        <small class="account-settings-header-subtitle"><%- gettext("These settings include basic information about your account. You can also specify additional information and see your linked social accounts on this page.") %></small>
+    <h3 class="header-title"><%- gettext("Account Settings") %></h3>
+        <ul class="left list-inline account-nav">
+            <% _.each(accountSettingsTabs, function(tab) { %>
+            <li class="account-nav-item">
+                <a data-name="<%- tab.name %>" class="account-nav-link <%- tab.class%>" href="#"><%- tab.label %></a>
```

Comment:
```a``` elements should just be used for links. If this is being used to capture click events it should be a ```button``` instead.